UNSIGNED_SWEEP.md

```diff
@@ -0,0 +1,43 @@
+# Unsigned Sweep Recovery Instructions
+
+Build an unsigned transaction using the user and backup public keys, independent of any BitGo services.
```

Comment:
This functionality is expected to be used only for cold wallets?

Reply:
I think yes based on the instructions in the unsigned sweep page in WRW. It is mentioned to enter user public key as mentioned on the recovery keycard. I think that is the case in the case of cold wallets only

Reply:
Please mention this is for cold wallets only in this case